src/components/application_manager/rpc_plugins/app_service_rpc_plugin/src/app_service_app_extension.cc

```diff
@@ -103,7 +103,8 @@ void AppServiceAppExtension::ProcessResumption(
     const smart_objects::SmartObject& subscriptions_app_services =
         resumption_data[app_service_info];
     for (size_t i = 0; i < subscriptions_app_services.length(); ++i) {
-      std::string service_type = resumption_data[i].asString();
+      std::string service_type =
+          resumption_data[app_service_info][i].asString();
```

Comment:
@YarikMamykin why don't you use `subscriptions_app_services` declared above?

Reply:
@AKalinich-Luxoft fixed in 12854ed786c7b982ed2a5a553f560a4cad66dd8d